jekyll/_cci2/hello-world-windows.md

```diff
@@ -11,7 +11,7 @@ version:
 - Server v2.x
 ---
 
-This document describes how to get started with continuous integration on **Windows build environments** on CircleCI. If this is your first time setting up CircleCI, we recommended checking out the [getting started guide]({{ site.baseurl}}/2.0/getting-started/).
+This document describes how to get started with continuous integration on **Windows execution environments** on CircleCI. If this is your first time setting up CircleCI, we recommend checking out the [getting started guide]({{ site.baseurl}}/2.0/getting-started/).
```

Comment:
```suggestion
This document describes how to get started with continuous integration on **Windows execution environments** on CircleCI. If this is your first time setting up CircleCI, we recommend checking out the [Getting Started guide]({{ site.baseurl}}/2.0/getting-started/).
```
Not sure if we have style guidelines specifically for whether links to pages or sections should be in Title or Sentence case, so I wanted to check here.

Reply:
Thank you @kbatuigas yes Title case for direct references like you say 🎉